.github/workflows/node.js.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [windows-latest, ubuntu-latest]
+        os: [ubuntu-latest]
         node-version: [14.x, 16.x, 18.x]
```

Comment:
While we're editing this file, may as well drop these old versions of Node.js, and add v20, which is the one we actually use